modules/efk-logging-about-fluentd.adoc

```diff
@@ -7,24 +7,16 @@
 
 {product-title} uses Fluentd to collect data about your cluster.
 
-Fluentd is deployed as a DaemonSet in {product-title} that deploys nodes according to a node
-label selector, which you can specify with the inventory parameter
-`openshift_logging_fluentd_nodeselector` and the default is `logging-infra-fluentd`.
-As part of the OpenShift cluster installation, it is recommended that you add the
-Fluentd node selector to the list of persisted node labels.
+Fluentd is deployed as a DaemonSet in {product-title} that deploys nodes to each OpenShift node.
 
 Fluentd uses `journald` as the system log source. These are log messages from
-the operating system, the container runtime, and OpenShift.
+the operating system, the container runtime, and {product-title}.
 
-The available container runtimes provide minimal information to identify the
-source of log messages. Log collection and normalization of logs can occur after
-a pod is deleted and additional metadata cannot be retrieved from the
-API server, such as labels or annotations.
-
-If a pod with a given name and project is deleted before the log collector
-finishes processing logs, there might not be a way to distinguish the log messages
-from a similarly named pod and project. This can cause logs to be indexed and
-annotated to an index that is not owned by the user who deployed the pod.
+The Docker and CRI-O runtimes provide minimal information to identify the source of log messages: project, pod name, 
```

Comment:
No Docker here, either.

Reply:
@kalexand-rh Are we just offering CRI-O in 4.0?

Reply:
Last I heard. Only CRI-O is supported with RHCOS, and the BYO team wasn't planning on carrying Docker support forward.